web/modules/custom/ct_user/ct_user.module

```diff
@@ -22,3 +23,47 @@ function ct_user_help($route_name, RouteMatchInterface $route_match) {
     default:
   }
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function ct_user_form_alter(&$form, $form_state, $form_id) {
+  // Check if the form is the user login form.
+  if ($form_id == 'user_login_form') {
+    // Add a custom validation function to the form.
+    $form['#validate'][] = 'ct_user_user_login_form_validate';
+  }
+}
+
+/**
+ * Custom validation function for the user login form.
+ */
+function ct_user_user_login_form_validate(&$form, $form_state) {
+  // Get the name value from the form.
+  $name = $form_state->getValue('name');
+
+  // Get the id of the user.
+  $ids = \Drupal::entityQuery('user')
+    ->condition('name', $name)
+    ->range(0, 1)
+    ->accessCheck()
```

Comment:
```suggestion
    ->accessCheck(FALSE)
```

I think access check should be disabled in this case as well. I know anonymous users can read user info but this will be useful in case that ever changes.